data/en/test_radio_mandatory_with_optional_other.json

```diff
@@ -0,0 +1,49 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.2",
+    "survey_id": "0",
+    "title": "Other input fields",
+    "theme": "default",
+    "legal_basis": "StatisticsOfTradeAct",
+    "description": "A questionnaire to demo the mandatory radio field.",
+    "groups": [{
+        "blocks": [{
+                "type": "Questionnaire",
+                "id": "radio-mandatory",
+                "questions": [{
+                    "type": "General",
+                    "id": "radio-mandatory-question",
```

Comment:
same as previous, more readable

Reply:
I've addressed this don't know why its still showing on git I haven't?